src/client/screens/UserAccountScreen.tsx

```diff
@@ -0,0 +1,319 @@
+                        resource={updateUserResource}
+                        inputProps={{
+                          value: data.email,
+                          required: true,
+                          type: "email"
+                        }}
+                        disabled={true}
+                      />
+                    </Flex>
+                  </Flex>
+                </Box>
+                <Box sx={style.subSection}>
+                  <Heading as="h3" sx={style.header}>
+                    Communications
+                  </Heading>
+                  <Flex>
+                    <Label sx={{ display: "inline-flex" }}>
+                      <Checkbox
+                        name="user-is-marketing-email-on"
+                        checked={data.isMarketingEmailOn}
+                        onChange={onCommunicationChange}
+                        disabled={isUpdatePending}
+                      />
+                      <Flex as="span" sx={{ textTransform: "none", fontWeight: "normal" }}>
+                        I want to receive product updates and announcements via email
+                      </Flex>
+                    </Label>
+                  </Flex>
+                  <RegisterTermsText />
+                </Box>
+                <Box sx={style.subSection}>
+                  <Button
+                    type="submit"
+                    sx={{ "&[disabled]": { opacity: "0.2" }, width: "130px" }}
+                    disabled={isSubmitDisabled}
+                  >
+                    {isUpdatePending ? <Spinner variant="styles.spinner.small" /> : "Save changes"}
+                  </Button>
+                </Box>
+              </Box>
+              <Divider />
+              <Box sx={style.section}>
+                <Heading as="h3" sx={{ mb: "3" }}>
+                  Password
+                </Heading>
+                <Box sx={style.textBox}>
+                  <Text>Forgot your password?</Text>{" "}
+                  <Themed.a
+                    as={RouterLink}
+                    to={{ pathname: `/forgot-password/`, state: { email: data.email } }}
+                    sx={{ cursor: "pointer", color: "blue.5" }}
+                  >
+                    Reset password
+                  </Themed.a>
+                  .
+                </Box>
+              </Box>
+              <Divider />
+              <Box sx={style.section}>
+                <Heading as="h3" sx={{ mb: "3" }}>
+                  Account deletion
+                </Heading>
+                <Box sx={style.textBox}>
+                  <Text>
+                    If you would like your account to be deleted, email{" "}
+                    <a
+                      href={`mailto:${accountDeleteEmail}`}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      sx={{ color: "blue.5" }}
+                    >
+                      districtbuilder@azavea.com
```

Comment:
Let's make use of the constant here too:
```suggestion
                      {accountDeleteEmail}
```